app/src/main/res/values/strings.xml

```diff
@@ -51,7 +51,7 @@
     <string name="option_wishlist_copy">Copy wishlist</string>
     <string name="option_wishlist_rename">Rename wishlist</string>
     <string name="option_wishlist_data_add">Create new wishlist</string>
-    <string name="option_wishlist_data_edit">Edit quantity</string>
```

Comment:
I'm not sure but this string might be reused in the actual wishlist where you want to actually Edit Quantity. If it is then just make separate strings for editing and renaming.